package.json

```diff
@@ -75,6 +75,7 @@
     "@babel/preset-env": "^7.5.4",
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.3.3",
+    "@firebase/app": "^0.4.16",
```

Comment:
Still a bit confused about this.  Shouldn't this be pulled in as one of the deps of the firebase package below?

Reply:
Notice that we already have it in `package-lock.json`, which means that something else is already pulling it in automatically.

Reply:
Installing `firebase@latest` and `firebase-admin@latest` removes `@firebase/app` from `package-lock.json`. Manually maintaining `package-lock.json` is probably not a good idea?

Reply:
We don't manually maintain it. Stuff gets added there when you run `npm install`.

So, it sounds like firebase no longer pulls in @firebase/app and they expect users to directly install that package?

Reply:
Correct.